### ISSUE TITLE
Officially support node v0.12 and node v4

### ISSUE THREAD

**Issue by edi9999** [Stuk/jszip#232]
It seems that the .travis.yml only tests on node v0.10. I think it is running fine with node v0.12 and node v4.

What is needed so that it works fine on node v4 and v0.12 ?

**Comment by edi9999** [comment]
It seems that they are many outdated packages : 

```
$ npm outdated

Package                    Current       Wanted  Latest  Location
qunit                        0.6.4        0.6.4   0.7.6  qunit
grunt-saucelabs              7.0.0        7.0.0   8.6.1  grunt-saucelabs
jshint                      2.5.11       2.5.11   2.8.0  jshint
grunt-browserify             2.1.4        2.1.4   4.0.1  grunt-browserify
grunt-contrib-jshint        0.10.0       0.10.0  0.11.3  grunt-contrib-jshint
grunt-contrib-uglify         0.4.1        0.4.1   0.9.2  grunt-contrib-uglify
browserify                  4.1.11       4.1.11  11.2.0  browserify
grunt-contrib-connect        0.7.1        0.7.1  0.11.2  grunt-contrib-connect
nopt                        1.0.10       1.0.10   3.0.4  grunt-cli > nopt
findup-sync                  0.1.3        0.1.3   0.3.0  grunt-cli > findup-sync
resolve                      0.3.1        0.3.1   1.1.6  grunt-cli > resolve
colors                       0.6.2        0.6.2   1.1.2  grunt > colors
dateformat             1.0.2-1.2.3  1.0.2-1.2.3  1.0.11  grunt > dateformat
async                       0.1.22       0.1.22   1.4.2  grunt > async
findup-sync                  0.1.3        0.1.3   0.3.0  grunt > findup-sync
glob                        3.1.21       3.1.21  5.0.14  grunt > glob
iconv-lite                  0.2.11       0.2.11  0.4.11  grunt > iconv-lite
minimatch                   0.2.14       0.2.14  2.0.10  grunt > minimatch
nopt                        1.0.10       1.0.10   3.0.4  grunt > nopt
rimraf                       2.2.8        2.2.8   2.4.3  grunt > rimraf
lodash                       0.9.2        0.9.2  3.10.1  grunt > lodash
which                        1.0.9        1.0.9   1.1.2  grunt > which
underscore.string            2.2.1        2.2.1   3.2.2  grunt > underscore.string
js-yaml                      2.0.5        2.0.5   3.4.2  grunt > js-yaml
async                       0.1.22       0.1.22   1.4.2  grunt > grunt-legacy-util > async
lodash                       0.9.2        0.9.2  3.10.1  grunt > grunt-legacy-util > lodash
underscore.string            2.2.1        2.2.1   3.2.2  grunt > grunt-legacy-util > underscore.string
which                        1.0.9        1.0.9   1.1.2  grunt > grunt-legacy-util > which
colors                       0.6.2        0.6.2   1.1.2  grunt > grunt-legacy-log > colors
underscore.string            2.3.3        2.3.3   3.2.2  grunt > grunt-legacy-log > underscore.string
lodash                       2.4.2        2.4.2  3.10.1  grunt > grunt-legacy-log > lodash
underscore.string            2.3.3        2.3.3   3.2.2  grunt > grunt-legacy-log > grunt-legacy-log-utils > underscore.string
colors                       0.6.2        0.6.2   1.1.2  grunt > grunt-legacy-log > grunt-legacy-log-utils > colors
lodash                       2.4.2        2.4.2  3.10.1  grunt > grunt-legacy-log > grunt-legacy-log-utils > lodash
coffee-script                1.3.3        1.3.3  1.10.0  grunt > coffee-script
```

**Comment by Mithgol** [comment]
...and Node.js v5.

**Comment by dduponchel** [comment]
Fixed by #233 and #269.
